pkgs/applications/video/mpv/scripts/mpris.nix

```diff
@@ -2,27 +2,33 @@
 
 stdenv.mkDerivation rec {
   name = "mpv-mpris-${version}.so";
-  version = "0.4";
+  version = "0.5";
 
   src = fetchFromGitHub {
     owner = "hoyon";
     repo = "mpv-mpris";
     rev = version;
-    sha256 = "1fr3jvja8s2gdpx8qyk9r17977flms3qpm8zci62nd9r5wjdvr5i";
+    sha256 = "07p6li5z38pkfd40029ag2jqx917vyl3ng5p2i4v5a0af14slcnk";
   };
 
   nativeBuildInputs = [ pkgconfig ];
 
   buildInputs = [ gobject-introspection mpv ];
 
+  # strip is not run as always in fixup phase because $out is a file and not a
+  # directory and mpris.so is not located in $out/lib. Hence, to make sure gcc
+  # is not referenced in the final output, we use `strip` like this instead of
+  # just copying it. See:
+  # https://discourse.nixos.org/t/debug-why-a-derivation-has-a-reference-to-gcc/7009
   installPhase = ''
-    cp mpris.so $out
+    strip --strip-debug mpris.so -o $out
```

Comment:
I would just recommend installing it at `$out/share/mpv/scripts` (looking at the [arch package](https://www.archlinux.org/packages/community/x86_64/mpv/), that is where scripts are installed) and adding `share/mpv/scripts` to `stripDebugList` list. And also asking upstream to make the installation directory configurable.

Of course mpv expression might need to be customized.

Reply:
That's a good idea also considering automatic updates (because if `$out` is `$name` and `pname` is not set @r-ryantm doesn't pick it) and it's done in latest push. Regarding patching upstream's Makefile, that's an even better idea but should we wait then for their response for this update? Or perhaps write the patch and add it to our own derivation?

Reply:
No need to block this on their response, we can always cherry pick the commit once they merge it.